### PR TITLE
[pangolin] MFTF Standalone Documentation

### PR DESCRIPTION
**Pull request by KevinBKozan**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

- Added a new mftf-standalone.md document with a basic how-to on how use MFTF as a completely standalone application.

whatsnew
Added guidelines how to [Set up a standalone MFTF](https://devdocs.magento.com/mftf/2.3/getting-started.html#set-up-a-standalone-mftf).

### CODE REVIEW

File: mftf/2.3/mftf-standalone.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+```bash
+$ git clone https://github.com/magento/magento2-functional-testing-framework.git
+```
+
+Run `composer install` at the MFTF root  and complete all normal setup steps using `bin/mftf` commands.
+
+File structure for standalone MFTF test files differs slightly from the Magento-embedded structure:
+
+* `.env` is located under `dev/.env`
```
Comment (dshevtsov) [inline mftf/2.3/mftf-standalone.md]: which [advanced configuration parameters](https://devdocs.magento.com/mftf/2.3/configuration.html) have to be set?

File: mftf/2.3/configuration.md [hunk 2]
```diff
@@ -163,22 +166,32 @@ Example:
...
+MAGENTO_BP
 TESTS_BP
 FW_BP
 TESTS_MODULES_PATH
 ```
 
+#### MAGENTO_BP
+
+The path to a local Magento codebase.
+It enables the [`bin/mftf generate`][mftf generate] commands to parse all modules at `app/code` and `vendor` of the provided Magento codebase.
```
Comment (KevinBKozan) [inline mftf/2.3/configuration.md]: It also enables the `bin/mftf run` commands to parse and run the tests.

File: mftf/2.3/configuration.md [hunk 3]
```diff
@@ -163,22 +166,32 @@ Example:
...
+
+```conf
+MAGENTO_BP=~/magento2/
+```
+
 #### TESTS_BP
 
 The path to where MFTF supplementary files are located in the Magento codebase.
-BP is an acronym for _BasePath_.
+BP is an acronym for _Base Path_.
```
Comment (KevinBKozan) [inline mftf/2.3/configuration.md]: I think it makes sense to move this line to right after the `### *_BP` header

Comment (dshevtsov) [reply mftf/2.3/configuration.md]: :ok:

File: mftf/2.3/configuration.md (hunk 2 above)
Comment (dshevtsov) [reply mftf/2.3/configuration.md]: will change to: 
It enables the [`bin/mftf`][mftf] commands such as `run` and `generate` to parse all modules of the Magento codebase for MFTF tests.

File: mftf/2.3/configuration.md [hunk 4]
```diff
@@ -26,6 +26,9 @@ Example:
 MAGENTO_BASE_URL=http://magento2.vagrant251
 ```
 
+{: .bs-callout .bs-callout-info }
+If the `MAGENTO_BASE_URL` contains a subdirectory (like `http://magento.test/magento2ce`), specify [`MAGENTO_CLI_COMMAND_PATH`][].
```
Comment (lorikrell) [inline mftf/2.3/configuration.md]: Should there be a link for []? Or should the [] be also in the code ticks?

File: mftf/2.3/getting-started.md [hunk 5]
```diff
@@ -67,26 +69,27 @@ git clone git@github.com:magento/magento2.git
 
 ### Step 2. Install dependencies {#install-dependencies}
 
-1. Checkout to the Magento version that you are going to test.
+Checkout to the Magento version that you are going to test.
```
Comment (lorikrell) [inline mftf/2.3/getting-started.md]: I would remove "to":   Checkout the Magento version that you are going to test.

File: mftf/2.3/getting-started.md [hunk 6]
```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
...
+### Step 7. Generate reports    {#reports}
 
 During testing, the MFTF generates test reports in CLI.
 You can generate visual representations of the report data using [Allure Framework][].
 To view the reports in GUI:
 
--  [install Allure][]
--  run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
+- [install Allure][]
+- run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
```
Comment (lorikrell) [inline mftf/2.3/getting-started.md]: May want to capitalize the first words in the bullets.

File: mftf/2.3/getting-started.md [hunk 7]
```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
...
 ```bash
 allure serve dev/tests/acceptance/tests/_output/allure-results/
 ```
 
 Learn more about Allure in the [official documentation][allure docs].
 
+## Set up a standalone MFTF
+
+The MFTF is a root level Magento dependency, but it is also available for use as a standalone application.
+The potential use case is when you develop the MFTF or contribute to it, as it facilitates debugging and tracking changes.
```
Comment (lorikrell) [inline mftf/2.3/getting-started.md]: Suggested edit: You may want to use a standalone application when you develop for or contribute to MFTF, which facilitates debugging and tracking changes.

File: mftf/2.3/getting-started.md [hunk 8]
```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
...
+In the `dev/.env` file, define the [basic configuration][] and [`MAGENTO_BP`][] parameters.
+
+### Step 5. Enable the Magento CLI commands {#add-cli-commands}
+
+Copy the `etc/config/command.php` file into your Magento installation at `<magento root directory>/dev/tests/acceptance/utils/`.
+Create the `utils/` directory, if you didn't find it.
+
+### Step 6. Remove the MFTF package dependency in Magento
+
+The MFTF uses the Magento `app/autoload.php` to read the Magento modules.
```
Comment (lorikrell) [inline mftf/2.3/getting-started.md]: Suggested edit: The MFTF uses the Magento `app/autoload.php` file to read Magento modules.

File: mftf/2.3/getting-started.md (hunk 5 above)
Comment (dshevtsov) [reply mftf/2.3/getting-started.md]: :ok:

File: mftf/2.3/getting-started.md (hunk 6 above)
Comment (dshevtsov) [reply mftf/2.3/getting-started.md]: :ok:

File: mftf/2.3/getting-started.md (hunk 7 above)
Comment (dshevtsov) [reply mftf/2.3/getting-started.md]: :ok:

File: mftf/2.3/getting-started.md (hunk 8 above)
Comment (dshevtsov) [reply mftf/2.3/getting-started.md]: :ok:

**Pull request merged**